train.py

```diff
@@ -99,9 +100,12 @@ def main(job_config: JobConfig):
         job_config.training.batch_size,
         job_config.training.seq_len,
         dp_degree,
-        dp_rank,
+        dp_rank
     )
 
+    # validation batch size
+    val_bs = job_config.validation.batch_size if job_config.validation.batch_size != 0 else job_config.metrics.batch_size
```

Comment:
See above comment, if the default is None this line becomes more readable
`val_bs = job_config.validation.batch_size if job_config.validation.batch_size else job_config.metrics.batch_size`